deploy/etc/crds/droplets.crd.yaml

```diff
@@ -68,3 +59,43 @@ spec:
       priority: 0
       JSONPath: .spec.provisiondelay
       description: Time to provision an object from creation
+    - name: Network
+      type: string
+      priority: 0
+      JSONPath: .spec.network
+      description: A droplet's physical connectivity, can be either "tenant" or "zgc"
+    - name: Zgc_id
+      type: string
+      priority: 0
+      JSONPath: .spec.zgc_id
+      description: ZGC id
+  validation:
+    openAPIV3Schema:
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            status:
+              type: string
+            mac:
+              type: array
+              items: string
+            ip:
+              type: array
+              items: string
+            interface:
```

Comment:
I don't think we need this one as we already have a spec.itf defined above.

Reply:
fixed